routes/conversations.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Router = require('express').Router;
+const Conversation = require('../models/Conversation');
+const Message = require('../models/Message');
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = Router();
+
+
+router.post('/', (req, res) => {
+
+  let conversation = new Conversation({
+    from_email: req.body.from_email,
+    to_email: req.body.to_email
+  });
+
+  conversation.save((err) => {
+    if (err.code === 11000){
```

Comment:
This code is bad because it's very unclear what it's actually checking.
Something like this would be much better: `if (err.code === MONGO_ERR_ALREADY_EXISTS)`

But ultimately, this code shouldn't even exist.  See above conversation.